### PR TITLE
[TIMOB-23128] Add missing local notification API's

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23128

### CODE REVIEW

File: iphone/Classes/TiAppiOSProxy.m [hunk 1]
```diff
@@ -566,6 +572,27 @@ -(id)scheduleLocalNotification:(id)args
 	if (badge!=nil) {
 		localNotif.applicationIconBadgeNumber = [TiUtils intValue:badge];
 	}
+    
+	id region = [args objectForKey:@"region"];
```
Comment (AngelkPetkov) [inline iphone/Classes/TiAppiOSProxy.m]: Add validation to ensure that region is an array.

**Comment by AngelkPetkov** [comment]
Left one comment, FT and CR approved otherwise! 👍

File: iphone/Classes/TiAppiOSProxy.m (hunk 1 above)
Comment (hansemannn) [inline iphone/Classes/TiAppiOSProxy.m]: It's an dictionary, will add the validation now.

**Comment by AngelkPetkov** [comment]
Nice Merging!

**Pull request merged**